controller/src/beerocks/master/tasks/channel_selection_task.cpp

```diff
@@ -1095,57 +1083,6 @@ void channel_selection_task::work()
         FSM_MOVE_STATE(GOTO_IDLE);
         break;
     }
-    case eState::SEND_HOSTAP_TX_ON_REQUEST: {
-
-        Socket *hostap_mac_sd = database.get_node_socket(hostap_mac);
-        auto request =
-            message_com::create_vs_message<beerocks_message::cACTION_CONTROL_HOSTAP_TX_ON_REQUEST>(
-                cmdu_tx);
```

Comment:
I notice that the monitor still has the concept of tx_enabled. Is that still relevant?

Reply:
The monitor notifies about TX change to the son_slave which influences the operational notification to the platform. If we remove the iface status notification we could remove it as well.